### ISSUE TITLE
OdooCompletionContributor StringIndexOutOfBoundsException

### ISSUE THREAD

**Issue by akshayk-patel** [wt-io-it/odoo-pycharm-plugin#103]
Plugin Version: 0.6.7
IntelliJ Version: PyCharm 2022.3
IntelliJ Build: PY-223.7571.203

<details><summary>java.lang.StringIndexOutOfBoundsException: begin 0, end -16, length 3</summary>

```
java.lang.StringIndexOutOfBoundsException: begin 0, end -16, length 3
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4604)
	at java.base/java.lang.String.substring(String.java:2707)
	at at.wtioit.intellij.plugins.odoo.OdooCompletionContributor.fillCompletionVariants(OdooCompletionContributor.java:23)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:81)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:246)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
this is fixed with 0.6.11 (pending jetbrains approval) if you are eager to test it you can use the jar file from: https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.11

**Comment by ap-wtioit** [comment]
0.6.11 is now available in the marketplace
